### PR TITLE
8.0 res fiscal position fix typo

### PR DESCRIPTION
**Pull request by gfcapalbo**
New PR for 8.0_res_fiscal_position, corrected a typo.
In the previous PR i pushed my feature branch to the main branch , therefore the PR appears merged.
Old PR  (  https://github.com/Therp/Therp-Addons/pull/31 ) .
Restarting here.

### CODE REVIEW

**Comment by gfcapalbo** [comment]
@StefanRijnhart  @NL66278  @lfreeke   reopened pull request here for 8.0 fiscal position with a typo fixed.   The old PR ( (  https://github.com/Therp/Therp-Addons/pull/31 )  looks merged because i pushed my feature branch on the main branch.     Please check this and approve for merge. Thanks.

**Comment by StefanRijnhart** [comment]
:+1:, thanks for the new PR.

File: res_partner_fiscal_position/model/res_country.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+class Country(models.Model):
+    """
+    adds a fiscal position to a country
+    """
+    _inherit = 'res.country'
+    property_account_position = fields.Many2one(
+        comodel_name='account.fiscal.position',
+        string="Default Fiscal Position",
+        company_dependent=True,
+        help="""The fiscal position will determine
```
Comment (hbrunn) [inline res_partner_fiscal_position/model/res_country.py]: this creates ugly spaces in your translation, replace by single quoted strings

File: res_partner_fiscal_position/model/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.onchange('country_id')
+    def get_country_fiscal_position(self):
+        """
+        when changing the partner's country update the
+        property_account_position to the one of that country
+        """
+        if not self.country_id:
```
Comment (hbrunn) [inline res_partner_fiscal_position/model/res_partner.py]: you don't need this check, just do the assignment below

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**